cpu/nrf51822/ng_ble/dev_ble_nrf.c

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (C) 2015 Jan Wagner <mail@jwagner.eu>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     dev_ble_nrf Low-Level Driver Inteface
+ * @{
+ *
+ * @file
+ * @brief       Low-level bluetooth low-energy driver for the NRF51822 radio interface
+ *
+ * @author      James Hollister <jhollisterjr@gmail.com>
+ * @author      Jan Wagner <mail@jwagner.eu>
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ * @author      Ludwig Ortmann <ludwig.ortmann@fu-berlin.de>
+ * @author      Martine Lenders <mlenders@inf.fu-berlin.de>
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
```

Comment:
Putting your name here would be enough. Even if it is a copied file, the logic (the essential stuff) is your work.